src/lib/alloc.c

```diff
@@ -109,7 +109,6 @@ static void write_pattern(struct mm_heap *heap_map, int heap_depth,
 			memset(
 				(void *)current_map->base, pattern,
 				current_map->count * current_map->block_size);
-					       sizeof(*current_map));
```

Comment:
wow, we must have other not compiled code hidden by conditional compilation.

Reply:
one more reason to try and make allyes and rand config builds working

Reply:
This is a result of running ```sed``` to remove the platform_commit() call.